doc/source/api.rst

```diff
@@ -1438,6 +1438,15 @@ Categorical Components
 
 .. _api.intervalindex:
 
+Interval
+--------
+
```

Comment:
Can you add properties and methods for these Scalars? (you *might* be able to get away with a directive to list all of them, for Period/Interval might be ok, Timestamp will show *everything* from its inheritied classes). If you can build the docs and show would be great (if there are *too* many things shown, we might just list some attributes / methods and point to the ``datetime.datetime`` docs otherwise.)

Can you also add for ``Timedelta``.

Reply:
![screenshot from 2017-09-07 02-07-13](https://user-images.githubusercontent.com/10288252/30149765-f06e1d7c-9376-11e7-8030-eb0d8e660ffe.png)
Ends up looking like above for `Timestamp` and `Period`. There's also room to clarify `days_in_month` vs `daysinmonth` as a compatibility alias. It looks like a lot of the existing properties don't need to be there - any guidance on this?